### PR TITLE
Fix crashreporter icon

### PR DESCRIPTION
**Pull request by dschmidt**

### CODE REVIEW

**Comment by guruz** [comment]
Did you try to build this via rotor?

**Comment by dschmidt** [comment]
It's the main purpose of this PR to fix things on Rotor.. :)

**Comment by dschmidt** [comment]
AppVeyor failure is just a hiccup... anongit.kde.org wasn't reachable :-|

File: src/gui/CMakeLists.txt [hunk 1]
```diff
@@ -216,12 +216,6 @@ endif()
 # list(APPEND CMAKE_MODULE_PATH ${ECM_MODULE_PATH})
 include(ECMAddAppIcon)
 
-# For historical reasons we can not use the application_shortname
```
Comment (ogoffart) [inline src/gui/CMakeLists.txt]: Maybe the comment could be moved to the other place.

**Pull request merged**